app/javascript/mastodon/emoji.js

```diff
@@ -1,36 +1,37 @@
 import { unicodeMapping } from './emojione_light';
 import Trie from 'substring-trie';
 
-const trie = new Trie(Object.keys(unicodeMapping));
+const excluded = ['®', '©', '™'];
+excluded.forEach(c => {
+  if (unicodeMapping[c] !== undefined) {
+    delete unicodeMapping[c];
```

Comment:
Remove the strings on the server (`emojione_light.js`) - no need to ship them if they are not used.